ledcontroller/__init__.py

```diff
@@ -440,3 +453,26 @@ def batch_run(self, *commands):
                 args = command[1:]
                 cmd(*args)
         self.repeat_commands = original_retries
+
+
+
+def rgb_to_hue(red, green, blue):
+    """ Convert RGB color to hue value.
+
+    0% lightness or 100% lightness (white & black),
+    (255, 255, 255) & (0,0,0) aren't representable.
+
+    Also note that the LimitlessLED color spectrum
+    starts at blue.
+
+    :param red: Red value (0-255).
+    :param green: Green value (0-255).
+    :param blue: Blue value (0-255).
+    :returns: Hue value (0-255).
+    """
+    hue = rgb_to_hls(red / 255, green / 255, blue / 255)[0] \
```

Comment:
Using `/ 255.0` is safer here, as python2 considers int / int as int (meaning, 100 / 255 = 0). This library should be compatible with both python2 and python3.